<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>494</width>
    <height>332</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>From:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>To:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="buttonExchange">
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="res.qrc">
        <normaloff>:/icons/double_arrows.png</normaloff>:/icons/double_arrows.png</iconset>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QDoubleSpinBox" name="spinBoxTo">
      <property name="decimals">
       <number>0</number>
      </property>
      <property name="minimum">
       <double>2.000000000000000</double>
      </property>
      <property name="maximum">
       <double>36.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QDoubleSpinBox" name="spinBoxFrom">
      <property name="decimals">
       <number>0</number>
      </property>
      <property name="minimum">
       <double>2.000000000000000</double>
      </property>
      <property name="maximum">
       <double>36.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="0" rowspan="2" colspan="3">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Translator</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="1">
        <widget class="QTextBrowser" name="textOutNumber"/>
       </item>
       <item row="0" column="0">
        <widget class="QTextEdit" name="textInNumber"/>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QPushButton" name="buttonTranslate">
         <property name="text">
          <string>Translate</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
   <zorder>buttonExchange</zorder>
   <zorder>spinBoxFrom</zorder>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>spinBoxTo</zorder>
   <zorder>verticalSpacer</zorder>
   <zorder>groupBox</zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
